src/DateTimePicker/StyledDateTimePicker.js

```diff
@@ -0,0 +1,22 @@
+import DateTimePicker from 'react-datetime';
+import styled from '@emotion/styled';
+import inputStyles from '../Input/inputStyles';
```

Comment:
I don't know if this is bad form, but I wanted to apply our input styles to their provided input. Works fine.

Reply:
Nope, this is being used exactly as intended. 👍